src/styles/glass.css

```
@@ -0,0 +1,840 @@
+  .tabbar-item:focus-visible,
+  .tabbar-item:active {
+    transition: color 160ms ease, opacity 160ms ease;
+    transform: none !important;
+  }
+
+  .tabbar-bubble {
+    transition: color 160ms ease, opacity 160ms ease, background 160ms ease, border-color 160ms ease;
+  }
+}
+
+@media (prefers-reduced-transparency: reduce) {
```

Biome (2.1.2)

[error] Don't use unknown media feature names.

Unexpected unknown media feature name.
You should use media feature names defined in the CSS Specifications.

(lint/correctness/noUnknownMediaFeatureName)